### PR TITLE
Adjusts radius text input alignment in appearance panel

### PR DESCRIPTION
**Pull request by DivyaPrabhakar**
References #3512

(Apologies for the UNREGISTERED sprawled over this image)

### CODE REVIEW

File: src/js/actions/search/stock.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+/*
```
Comment (iwehrman) [inline src/js/actions/search/stock.js]: I don't think this is supposed to be here :grimacing:

**Comment by iwehrman** [comment]
Also, I recommend checking with @tomwmcrae about buying a copy of [xScope](http://xscopeapp.com/). It's an essential tool for careful UI work, like you're doing. I think he can be convinced.

**Comment by iwehrman** [comment]
Looks amazing, btw!

**Comment by DivyaPrabhakar** [comment]
@iwehrman  oops! Back to you.

**Comment by iwehrman** [comment]
Beautiful!

**Comment by tomwmcrae** [comment]
@DivyaPrabhakar: Sure. Will you also need an Apple Watch to get the full value out of this?

From [xScope](http://xscopeapp.com/) page:
Mirror   Effortlessly see your designs and applications on iOS and Apple Watch while you work on them from your Mac.

**Comment by DivyaPrabhakar** [comment]
@tomwmcrae You said it, not me. I'll expect it on my desk tomorrow. #bestmanagerever

**Pull request merged**